addons/mail/static/src/models/thread/thread.js

```diff
@@ -1192,6 +1256,20 @@ function factory(dependencies) {
             return [['replace', this.messages.filter(message => message.isNeedaction)]];
         }
 
+        /**
+         * @returns {integer|undefined}
+         * @private
+         */
+        _computeNewMessageSeparatorId() {
+            if (!this.lastSeenByCurrentPartnerMessageId) {
+                return;
+            }
```

Comment:
I think it should return `0` instead (or `clear()`, depending on what we consider the default value). Returning `undefined` means no change means it could keep an obsolete value.